.rubocop.yml

```diff
@@ -59,6 +64,18 @@ Style/BlockDelimiters:
   Enabled: true
   EnforcedStyle: line_count_based
 
+Style/SignalException:
+  EnforcedStyle: semantic
+
+Style/TrailingCommaInLiteral:
+  EnforcedStyleForMultiline: consistent_comma
```

Comment:
I'm personally not a fan of the trailing comma. But I guess if that's what the standard is now a days...